learn_django/trip_project/views.py

```diff
@@ -48,7 +48,7 @@ def create_profile(request):
         if user_form.is_valid():
             user = user_form.save()
             profile = Profile.objects.get(user=user)
-            profile_form = create_profile(request.POST, instance=profile)
+            profile_form = Create_Profile_Form(request.POST, instance=profile)
```

Comment:
`CreateProfileForm` class name should be in all caps read more about casing in python.